packages/serverpod/lib/src/server/websocket_request_handlers/method_websocket_request_handler.dart

```diff
@@ -391,7 +413,44 @@ class _MethodStreamManager {
     WebSocket webSocket,
     Session session,
     SerializationManager serializationManager,
+    _RequiredAuthentication? requiredAuthentication,
   ) {
+    late void Function(SerializableModel) authenticationRevokedCallback;
+
+    if (requiredAuthentication != null) {
+      // If authentication was required, a listener is registered that closes
+      // the stream if the authentication token is revoked.
+      authenticationRevokedCallback = (event) async {
+        var authenticationRevoked = switch (event) {
```

Comment:
Suggestion:  As mentioned elsewhere, make it explicit that it's a boolean to not confuse with other objects
```suggestion
        var authenticationWasRevoked = switch (event) {
```